sunpy/database/database.py

```diff
@@ -857,6 +857,29 @@ def add_from_vso_query_result(self, query_result,
                 query_result, self.default_waveunit),
             ignore_already_added)
 
+    def add_from_fido_search_result(self, search_result,
+                                    ignore_already_added=False):
+        """
+        Generate database entries from a Fido search result and add all the
+        generated entries to this database.
+
+        Parameters
+        ----------
+        search_result : `sunpy.net.dataretriever.downloader_factory.UnifiedResponse`
```

Comment:
this is not the FQN of this class anymore it's now `sunpy.net.fido_factory.UnifiedResponse`